README.md

```diff
@@ -0,0 +1,11 @@
+Name: Cruise Ship
```

Comment:
You should always have a heading element (made with a single #) at the start of your readme containing your title. Remember your README is the first thing anybody sees when looking at your repo, so it should have all the information needed to install and use your app as well as an explanation of what it does.

Some things you should include/do:
- what technologies are used,
- A description of what the app actually does. Does it work for all cities in all countries? What range does it show?

Also useful is [markdown live preview](https://markdownlivepreview.com/) so you can play with what the file will look like without having to do multiple commits to the repo to see the results, have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too